src/test/java/serenity/bdd/steps/DefinitionSteps.java

```diff
@@ -50,4 +51,18 @@ public void thenTheyShouldSeeADefinitionInTable(String definition) {
         endUser.should_see_definition(definition);
     }
 
+    @Given("system is clean")
+    public void systemIsClean(){
+        endUser.cleanup();
+    }
+
+    @When("the user checks pet by status")
```

Comment:
Create more re-usable step. 
Add parameter $status `the user checks pet by status $status`